deployment/production/docker/inasafe-headless/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN pip install -r /REQUIREMENTS-realtime.txt
 # Copy ubuntu fonts
 RUN apt-get -y --force-yes install wget unzip
 ADD ubuntu-font-family-0.83.zip /ubuntu-font-family-0.83.zip
-RUN mv fad7939b-ubuntu-font-family-0.83.zip ubuntu-font-family-0.83.zip
+RUN mv ubuntu-font-family-0.83.zip ubuntu-font-family-0.83.zip
```

Comment:
So the `mv` lost a lot of interest now ;-)

Reply:
;) ... I tried to build it using cloud docker to check. I was hoping it could build quickly. But it was faster to build it myself without having to queue.